pkgs/development/compilers/unison/default.nix

```diff
@@ -31,14 +33,18 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optional (!stdenv.isDarwin) autoPatchelfHook;
-  buildInputs = lib.optionals (!stdenv.isDarwin) [ ncurses6 zlib gmp ];
+  buildInputs = lib.optionals (!stdenv.isDarwin) [ gmp ncurses6 zlib ];
 
   installPhase = ''
     mkdir -p $out/bin
-    mv ucm $out/bin
+    mv runtime $out/runtime
```

Comment:
I think conventionally this should go in `$out/lib` or `$out/lib/runtime`.

Reply:
Actually, I see [a bunch of derivations actually use `libexec`](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20out%2Flibexec&type=code), but the Nixpkgs manual says [put them in `lib`](https://nixos.org/manual/nixpkgs/stable/#outputlib).

Reply:
Either `out` or `lib` yeah, let me update it to `lib`.